chemprop/nn/loss.py

```diff
@@ -27,12 +28,12 @@
 
 
 class LossFunction(nn.Module):
-    def __init__(self, task_weights: Tensor):
+    def __init__(self, task_weights: ArrayLike = 1.0):
         """
         Parameters
         ----------
-        task_weights : Tensor
-            a tensor of shape `t` or `1 x t` containing the per-task weight.
+        task_weights :  ArrayLike = 1.0
+            the per-task weights of shape `t` or `1 x t`. Defaults to all tasks having a weight of 1.
         """
         super().__init__()
         task_weights = torch.as_tensor(task_weights, dtype=torch.float32).view(1, -1)
```

Comment:
```suggestion
        task_weights = torch.as_tensor(task_weights, dtype=torch.float).view(1, -1)
```

unless you have good reason to specify precision, prefer high-level aliases (same goes for numpy, c.f. `np.single` vs. `np.float32`)